datacenter/ucp/2.1/guides/user/services/use-domain-names-to-access-services.md

```diff
@@ -95,8 +95,8 @@ to your services.
 
 The HTTP routing mesh can route to a service, as long as that service:
 
-* Is attached to a network that has the `com.docker.ucp.mesh.http` label. You
-can use the default `ucp-hrm` network or create your own
+* Is attached to a network that has the `com.docker.ucp.mesh.http` label. Enabling the 
```

Comment:
I don't think this addresses #2580. For that, I think you should had a snippet after line 181 that shows how to create an overlay network with the HRM labels and UCP permission labels.

You also need to mention that if you're creating a a new HRM network you need to disable the HRM service first, or disable and enable the HRM service after you create the network. And doing that, requires admin permissions.